hint.css

```diff
@@ -650,3 +650,89 @@
 .hint--no-shadow:before, .hint--no-shadow:after {
   text-shadow: initial;
   box-shadow: initial; }
+
+  /*Below codes are for the pre-loader contributed by Ankur Bodhak*/
+  *,
```

Comment:
This will affect complete page because of just your loader. Changes should be localised to your addition only ideally